app/reducers/imageGallery.ts

```diff
@@ -0,0 +1,32 @@
+import { produce } from 'immer';
+import { createSelector } from 'reselect';
+import createEntityReducer, { fetching } from 'app/utils/createEntityReducer';
+import { ImageGallery, File } from '../actions/ActionTypes';
+
+export default createEntityReducer({
+  key: 'imageGallery',
+  types: {
+    fetch: ImageGallery.FETCH_ALL,
+  },
+  mutate: (state, action) => {
+    switch (action.type) {
+      case File.PATCH.SUCCESS:
+        if (action.meta.body?.save_for_use === false) {
+          return produce(state, (draft) => {
+            draft.items = draft.items.filter((id) => id !== action.meta.id);
+          });
+        }
+        break;
+    }
+    return state;
+  },
+  /*   schema: imageGallerySchema,
+  fetching: fetching(ImageGallery.FETCH_ALL),
+ */
+});
+export const selectImageGalleries = createSelector(
```

Comment:
```suggestion
export const selectImageGalleryEntries = createSelector(
```
Just to be consistent with our language here, I understand it as one image gallery with multiple images (or entries?) inside. 
It doesn't really matter what we call things, but it should be consistent.